src/storage-manager.coffee

```diff
@@ -149,6 +149,9 @@ class _StorageManager
 
     if @expectations?
       @handleExpectations(loader)
+      loader.loadOptions.returnValues ?= {}
```

Comment:
Sometimes the `returnValues` is actually populated. However, expectations are short-circuiting the code which mixes in the jQuery XHR object into the promise / return values. 

So this just ensures the same "code" happens when it uses expectations.